intelmq/bots/outputs/xmpp/output.py

```diff
@@ -1,4 +1,24 @@
-# -*- coding: utf-8 -*-
+"""
+XMPP Output Bot
+Connects to a XMPP Server and sends data to a user.
+
+TODO: Introduce Multi User Chat like in XMPP Collector
+
+Requires Python >= 3.4
+Requires sleekxmpp >= 1.0.0-beta5
```

Comment:
> > 1.0.0-beta5 does not exist on pypi.
> But on Ubuntu 14.04 Repositories.
Using `1.0~beta5-2` from trusty:
```python
>>> sleekxmpp.exceptions.IqError
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'IqError'
>>> sleekxmpp.__version__
'1.0beta5'
```
using `1.0beta6.1` via pip:
```python
>>> sleekxmpp.exceptions.IqError
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'IqError'
>>> sleekxmpp.__version__
'1.0beta6.1'
```
`1.0rc1` is the first version that works for me.